code/modules/mob/living/simple_animal/bot/atmosbot.dm

```diff
@@ -189,7 +189,7 @@
 			return
 
 /mob/living/simple_animal/bot/atmosbot/proc/try_speak(message)
-	if (has_spoken || last_speech > world.time + 3 MINUTES)
+	if (has_spoken || IS_TIME_FUTURE(last_speech + 3 MINUTES))
```

Comment:
Unless I understood this incorrectly, I found this is an error. `last_speech` will never be more than current world.time.